src/server/webkit.ts

```diff
@@ -203,7 +205,7 @@ export class WebKit implements BrowserType {
 
     const defaultOptions = {
```

Comment:
Should we follow up with removing BrowserFetcherOptions? We do not use anything but default.

Reply:
Sure, sg